lib/ansible/modules/slurp.py

```diff
@@ -99,20 +98,18 @@ def main():
 
     try:
         with open(source, 'rb') as source_fh:
-            source_content = source_fh.read()
-    except (IOError, OSError) as e:
-        if e.errno == errno.ENOENT:
-            msg = "file not found: %s" % source
-        elif e.errno == errno.EACCES:
-            msg = "file is not readable: %s" % source
-        elif e.errno == errno.EISDIR:
-            msg = "source is a directory and must be a file: %s" % source
+            data = base64.b64encode(source_fh.read())
+    except OSError as ex:
+        if ex.errno == errno.ENOENT:
+            msg = f"file not found: {source!r}"
+        elif ex.errno == errno.EACCES:
+            msg = f"file is not readable: {source!r}"
+        elif ex.errno == errno.EISDIR:
+            msg = f"source is a directory and must be a file: {source!r}"
```

Comment:
If we're leaving the filename at the end of the message, then it shouldn't be quoted:

```suggestion
        if ex.errno == errno.ENOENT:
            msg = f"File not found: {source}"
        elif ex.errno == errno.EACCES:
            msg = f"File is not readable: {source}"
        elif ex.errno == errno.EISDIR:
            msg = f"Source is a directory and must be a file: {source}"
```

Alternatively, the messages could be reworded to quote the source. For example:

```python
f"Source {source!r} is a directory and must be a file."
```